code/game/objects/items/weapons/tools/misc.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/tool/omnitool
 	name = "Asters \"Munchkin 5000\""
-	desc = "Fuel powered monster of a tool. Its weldier part is most advanced one, capable of welding things without harmfull glow and sparks, so no protection needed."
+	desc = "Fuel powered monster of a tool. Its welder part is most advanced one, capable of welding things without harmfull glow and sparks, so no protection needed."
```

Comment:
"A fuel powered monster of a tool. Its welding attachment is capable of welding things without an eye-damaging flash, so no eye protection is required."